data.js

```diff
@@ -0,0 +1,33 @@
+const data = [
+    {
+        id: 1,
+        type: 'toro',
+        number: 11,
+        milk: false,
+        cuernos: 'No. of Horns'
+
+
+    },
+    {
+        id: 2,
+        type: 'vaca',
+        number: 22,
+        milk: true,
+        cuernos: 'No. of Horns'
```

Comment:
What we mean by something that would go into a dropdown is kind of like with the pokedex where you can categorize the items. 
An example of something you could do would be to create something like animal_type and then create categories like "mammal", "reptile", and "amphibian". Then when you click on a dropdown from the front end later you'd choose between mammal, reptile, and amphibian to filter the data.